test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightDeltaWriterTests.cs

```diff
@@ -2137,7 +2137,7 @@ public void WriteNestedDeletedResourceIn40ShouldFail(bool isResponse)
                 writer.WriteDeltaDeletedEntry(new ODataDeltaDeletedEntry("Products/1", DeltaDeletedEntryReason.Deleted));
             };
 
-            writeAction.Throws<ODataException>(Strings.ODataWriterCore_InvalidTransitionFromExpandedLink("NestedResourceInfoWithContent", "DeletedResource"));
+            writeAction.Throws<ODataException>(Strings.ODataWriterCore_NestedContentNotAllowedIn40DeletedEntry);

```

Comment:
Why are we still throwing this exception?
Is the behaviour similar to v4.01?

Reply:
this will be in a different pr to fix delta deleted resources in v4.0. This PR just solves nested delta resource sets.